sitewit/models.py

```diff
@@ -113,3 +119,30 @@ def delete(cls, account_token):
         result = cls.get_service().delete_account(account_token)
 
         return Account(result)
+
+
+class Subscription(SiteWitServiceModel):
+    def __init__(self, site_id, url, data):
+        self.site_id = site_id
+        self.url = url
+        self.ad_spend = Decimal(data['budget'])
+        self.price = Decimal(data['fee'])
+        self.campaign_id = data['campaignId']
+        self.currency = data['currency']
+
+    @classmethod
+    def iter_subscriptions(cls):
+        service = cls.get_service()
+        limit = 100
+
+        for offset in itertools.count(0, limit):
```

Comment:
I prefer `for .. in` to `while True`. Maybe `range(limit)` instead of `itertools.count` would make this more readable?